File: C++/Crud.cpp
class Crud{
    
    private:
    // Membuat sebuah vector 'list' bertipe Mahasiswa yang nantinya digunakan untuk menampung data mahasiswa
        vector<Mahasiswa> list;

    public:
    // konstruktor
    Crud(){
    }

    // method untuk menambahkan data mahasiswa ke dalam vector 'list'
    void menambahData(Mahasiswa data){
        list.push_back(data);
    }

    // method untuk melihat seluruh data mahasiswa yang ada di dalam vector 'list'
    void melihatData(){
        if(list.size() == 0){ // jika tidak ada data di dalam vector 'list'
            cout << "Data Mahasiswa Kosong" << endl;
        }
        else{ // jika ada data di dalam vector 'list'
            cout << "DATA MAHASISWA" << endl;

            // looping untuk mengambil data dari seluruh elemen vector 'list'
            for(int i=0; i<list.size(); i++){
                cout << "===============================================" << endl;
                cout << "NIK            : " << list[i].get_NIK() << endl;
                cout << "Nama           : " << list[i].get_Name() << endl;
                cout << "Jenis Kelamin  : " << list[i].get_Gender() << endl;
                cout << "Universitas    : " << list[i].get_Asal_univ() << endl;
                cout << "Email          : " << list[i].get_Email() << endl;
                cout << "NIM            : " << list[i].get_Nim() << endl;
                cout << "Fakultas       : " << list[i].get_Fakultas() << endl;
                cout << "Jurusan        : " << list[i].get_Jurusan() << endl;
            }
            cout << "===============================================\n" << endl;
        }
    }

        // Destruktor
        ~Crud(){
        }
    
};

File: Python/Crud.py
class Crud:
    
    def __init__(self):
        # Membuat sebuah list kosong yang nantinya digunakan untuk menampung data mahasiswa
        self.list = []
    
    # method untuk menambah data mahasiswa yang ditambahkan user ke dalam list
    def menambahdata(self, data):
        self.list.append(data)
        
    # method untuk melihat seluruh data mahasiswa yang ada di dalam list
    def melihatdata(self):
        if len(self.list) == 0:  # jika tidak ada data di dalam list
            print("\nData Mahasiswa Kosong")
        else: # jika ada data di dalam list
            print("                 DATA MAHASISWA")
            # looping untuk mengambil data dari seluruh elemen list
            for i in range(len(self.list)):
                print("===============================================")
                print("NIK            : ", self.list[i].get_NIK())
                print("Nama           : ", self.list[i].get_Name())
                print("Jenis Kelamin  : ", self.list[i].get_Gender())
                print("Universitas    : ", self.list[i].get_Asal_univ())
                print("Email          : ", self.list[i].get_Email())
                print("NIM            : ", self.list[i].get_Nim())
                print("Fakultas       : ", self.list[i].get_Fakultas())
                print("Jurusan        : ", self.list[i].get_Jurusan())
                print("===============================================\n")
        